src/internal.c

```diff
@@ -6900,17 +6900,19 @@ static int wolfSSLReceive(WOLFSSL* ssl, byte* buf, word32 sz)
                 ssl->options.isClosed = 1;
                 return -1;
 
-            #ifdef WOLFSSL_DTLS
             case WOLFSSL_CBIO_ERR_TIMEOUT:
-                if (IsDtlsNotSctpMode(ssl) &&
+            #ifdef WOLFSSL_DTLS
+                if (ssl->options.dtls &&
+                    IsDtlsNotSctpMode(ssl) &&
```

Comment:
Doesn't IsDTLSNotSctpMode already check for options.dtls?

Reply:
Fix pushed. You were correct. `IsDtlsNotSctpMode` already checks for if its DTLS.